zproject/prod_settings_template.py

```diff
@@ -72,10 +72,12 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 # The email From address to be used for automatically generated emails
-DEFAULT_FROM_EMAIL = "Zulip <zulip@example.com>"
+# (e.g. "Zulip <zulip@example.com>")
+DEFAULT_FROM_EMAIL = "Zulip <zulip@{0}>".format(EXTERNAL_HOST.split(":")[0])
 # The noreply address to be used as Reply-To for certain generated emails.
 # Messages sent to this address should not be delivered anywhere.
-NOREPLY_EMAIL_ADDRESS = "Zulip <noreply@example.com>"
+# (e.g. "Zulip <noreply@example.com>")
```

Comment:
I think we usually use `zulip.example.com` as the domain name in examples, so we should probably follow that pattern.